sdk/core/azure-core/test/ut/CMakeLists.txt

```diff
@@ -39,10 +39,8 @@ endif()
 
 include(GoogleTest)
 
```

Comment:
We should not take a dependency on OpenSSL on WIndows if at all possible. The MSRC guidance for using cryptography on Windows indicates that OpenSSL should NOT be used.